### DISCUSSION TITLE
ADMINISTRATOR ROLE

### DISCUSSION THREAD

**Discussion by Amailson** [Authenticator-Extension/Authenticator#1100] [Ideas]
I would like to propose an administrator version. Where it is possible to create a password to enable and disable the backup or sharing of passwords between users, and it would still be interesting that this administrator can even block the removal of the extension in the browser. Otherwise, the extension is perfect and fluid.

**Comment by mymindstorm** [comment]
We already support settings like this through enterprise policy: https://authenticator.cc/docs/en/policy-config

**Comment by Amailson** [comment]
Can I ask a question? So from what I understand there is no option within
the extension itself. But it is possible through a group policy without own
windows

Can I ask a question? So from what I understand there is no option
within the extension itself. But it is possible through a group policy
w


Em qua., 30 de ago. de 2023 às 12:18, Brendan ***@***.***>
escreveu:

> We already support settings like this through enterprise policy:
> https://authenticator.cc/docs/en/policy-config
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Authenticator-Extension/Authenticator/discussions/1100#discussioncomment-6865697>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A46RFX27W5MYANBJ2RN2QJDXX5KURANCNFSM6AAAAAA4ERIJY4>
> .
> You are receiving this because you authored the thread.
>


Can I ask a question? So from what I understand there is no option within
the extension itself. But it is possible through a group policy without own
windows.

**Reply by mymindstorm** [reply]
Yes
